content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -73,3 +73,37 @@ Example: `authorization.k8s.io/decision: "Human-readable reason for the decision
 This annotation gives reason for the [decision](#authorization-k8s-io-decision) in Kubernetes audit logs.
 
 See [Auditing](/docs/tasks/debug-application-cluster/audit/) for more information.
+
+## missing-san.invalid-cert.kubernetes.io/$hostname
+
+Example: `missing-san.invalid-cert.kubernetes.io/example-svc.example-namespace.svc: "relies on a legacy Common Name field instead of the SAN extension for subject validation"`
+
+Used by Kubernetes version v1.24 and later
+
+This annotation indicates an admission or conversion webhook or aggregated API server
+is using an invalid certificate that is missing `subjectAltNames`.
```

Comment:
What about other static webhooks? Like authz, audit, token authenticator, etc?

Reply:
I checked and those are included as well... broadened this